src/player.cpp

```diff
@@ -14149,12 +14142,26 @@ int player::get_stamina_max() const
 
 void player::burn_move_stamina( int moves )
 {
+    int overburden_percentage = 0;
+    if (weight_carried() > weight_capacity()) {
```

Comment:
It would be a good idea to save `weight_carried()` and `weight_capacity()` to temporary variables, to avoid recalculating them few times.